skafka/src/main/scala/com/evolutiongaming/skafka/consumer/Consumer.scala

```diff
@@ -369,6 +370,13 @@ object Consumer {
           }
         }
 
+        def subscribe(topics: Nes[Topic], listener: RebalanceListener1[F]) = {
+          // TODO derive ToTry timeout based on ConsumerConfig.maxPollInterval: FiniteDuration (defaulf of 5.minutes)
```

Comment:
I propose to not bother at all with this right now.

Reply:
will create a github issue and add link in todo comment

Reply:
done - a38c6c7ce9e8bdde1279f1c41894adecbb3ee914